src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -9,6 +9,7 @@
 #include "graph_iterator_proto_txt.hpp"
 #include "graph_iterator_saved_model.hpp"
 #include "helper_ops/internal_operation.hpp"
+#include "helper_ops/keep_in_graph_op.hpp"
```

Comment:
not needed, please include header where it is really used.

Reply:
Removed.